source/blog/2018-07-27-the-ember-times-issue-57.md

```diff
@@ -14,7 +14,7 @@ Read either on the [Ember blog](https://www.emberjs.com/blog/2018/07/27/the-embe
 
 
 ## [Ember Inspector - Help Shape The Future!](https://github.com/emberjs/ember-inspector)
-We're pushing hard to make Ember Inspector the world class dev tools you deserve, and would love to get your input and help!
+We're pushing hard to make Ember Inspector the world class DevTools you deserve, and would love to get your input and help!
```

Comment:
I still think this should read in the singular and lower case (devtool), but I will leave this to @emberjs/the-ember-times-editors to decide a final edit.